Cannot find module ./MessageBuilder

It runs on windows perfectly fine but using ubuntu rn. 


Error: Cannot find module './MessageBuilder'
Require stack:
- /root/Snooful-master/node_modules/discord-webhook-node/src/classes/webhook.js
- /root/Snooful-master/node_modules/discord-webhook-node/src/index.js
- /root/Snooful-master/src/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/root/Snooful-master/node_modules/discord-webhook-nod                                                                                                                                                             e/src/classes/webhook.js:2:24)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/root/Snooful-master/node_modules/discord-webhook-nod                                                                                                                                                             e/src/index.js:1:17)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/root/Snooful-master/src/index.js:40:17)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js                                                                                                                                                             :71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/root/Snooful-master/node_modules/discord-webhook-node/src/classes/webhook.                                                                                                                                                             js',
    '/root/Snooful-master/node_modules/discord-webhook-node/src/index.js',
    '/root/Snooful-master/src/index.js'

--- a/package.json
+++ b/package.json
@@ -1,9 +1,12 @@
 {
   "name": "discord-webhook-node",
-  "version": "1.1.7",
+  "version": "1.1.8",
   "description": "Allows for easy webhook sending through discord's webhook API",
   "main": "./src/index.js",
-  "exports": "./src/index.mjs",
+  "exports": {
+    "import": "./src/index.mjs",
+    "require": "./src/index.js"
+  },
   "typings": "index.d.ts",
   "scripts": {
     "test": "mocha --timeout 50000"

